frontend/src/components/AboutUs.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="cfp-background">
+        <v-container fluid text-xs-center>
+            <v-layout row class="cfp-deco-title cfp-font-title">
```

Comment:
This is responsible for the horizontal line below the title I guess. Just wanted to not that the padding between title and horizontal line isn't symmetrical to the padding below the horizontal line so that "Kim jesteśmy" and "Co robimy" are just below the line. It's nothing signigicant but it bothers my pedantic self :D